Codegen/src/Generators/MixedTransformGenerator.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TypeCobol.Codegen.Skeletons;
+using TypeCobol.Compiler;
+using TypeCobol.Compiler.Nodes;
+using TypeCobol.Compiler.Text;
+
+namespace TypeCobol.Codegen.Generators
+{
+    public class MixedTransformGenerator : DefaultGenerator
```

Comment:
I'd rather use a GeneratorWrapper instead to inherit from DefaultGenerator.
This will allow to combine the MixedTransformGenerator with any other Generator.